Pinta.Docking/Pinta.Docking.dll.config

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<dllmap os="!windows,osx" dll="libglib-2.0-0.dll" target="libglib-2.0.so.0"/>
```

Comment:
Is this GTK2 stuff that was copied over from the master branch, or is it still needed?